app.py

```
@@ -986,23 +1063,28 @@ def _pixel_test_commands(
     chase_index: int = 0,
     chase_rgb: Optional[Tuple[float, float, float]] = None,
```

Ruff (0.16.1)

[warning] Unused function argument: `chase_rgb`

(ARG001)